web/src/pages/Resources.tsx

```diff
@@ -89,7 +92,7 @@ const Resources = () => {
                       <div key={timestamp} className="w-full flex flex-row justify-start items-start">
                         <div className="w-16 sm:w-24 pt-4 sm:pl-4 flex flex-col justify-start items-start">
                           <span className="text-sm opacity-60">{date.getFullYear()}</span>
-                          <span className="font-medium text-xl">{date.toLocaleString("default", { month: "short" })}</span>
+                          <span className="font-medium text-xl">{date.toLocaleString(locale, { month: "short" })}</span>
```

Comment:
We can use `i18n.language` directly.

```suggestion
                          <span className="font-medium text-xl">{date.toLocaleString(i18n.language, { month: "short" })}</span>
```

Reply:
yep, will change and update the PR

Reply:
are you suggesting not using `findNearestLanguageMatch` and directly using `i18n.language` instead? or just replacing `locale` with ` findNearestLanguageMatch(i18n.language)` is fine?

I saw in other modules we are using `findNearestLanguageMatch`, so used it.